js/Cargo.toml

```diff
@@ -14,3 +14,6 @@ serde =  { version = "1.0", features = ["derive", "rc"] }
 serde_json = "1.0"
 adblock = { path = "../", features = ["css-validation", "content-blocking", "resource-assembler"] }
 neon = { version = "^0.10.1", default-features = false, features = ["napi-1"] }
+
+[features]
+default-panic-hook = []
```

Comment:
I don't think this does anything?

Reply:
One of the changes in Rust 1.84 is https://github.com/rust-lang/rust/pull/132577, so this is now a lint warning (and thus an error since we've configured lint warnings as errors). I think it's resolved in newer versions of Neon, but we can leave it here as-is for now.